jbmc/src/java_bytecode/java_types.cpp

```diff
@@ -582,26 +582,25 @@ typet java_type_from_string(
         std::back_inserter(parameters),
         [](const typet &type) { return java_method_typet::parametert(type); });
 
-      return java_method_typet(std::move(parameters), std::move(return_type));
+      return java_method_typet(std::move(parameters), std::move(*return_type));
```

Comment:
Are we sure this has a value?

Reply:
I have no idea, but it was previously used without any sanity check, so I'd now rely on `operator*` failing an assertion...